alcs-api/src/application/application-subtask/application-subtask.controller.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { classes } from '@automapper/classes';
+import { AutomapperModule } from '@automapper/nestjs';
+import { createMock, DeepMocked } from '@golevelup/nestjs-testing';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ClsService } from 'nestjs-cls';
+import { ApplicationSubtaskProfile } from '../../common/automapper/application-subtask.automapper.profile';
+import { ApplicationProfile } from '../../common/automapper/application.automapper.profile';
+import { mockKeyCloakProviders } from '../../common/utils/test-helpers/mockTypes';
```

Comment:
Little thing, but why isn't this upper camel case?

Reply:
Its a variable, not a type or class.